golem_messages/validators.py

```diff
@@ -13,7 +13,7 @@ def validate_varchar(field_name, value, max_length):
 
 
 def validate_integer(field_name, value):
-    if not isinstance(value, int):
+    if not isinstance(value, int) and value is not None:
```

Comment:
@kbeker I'd just leave the previous check in place and treat `None` as illegal ... it should be the responsibility of the code that creates this message to ensure that it always sets the price... 

btw, that's e.g. why I had suggested, in one of my previous comments (https://github.com/golemfactory/golem-messages/pull/179#issuecomment-376936222) to add this property to the factory (here ->
https://github.com/golemfactory/golem-messages/blob/master/tests/factories.py#L36) so any tests that rely on factory-provided `TaskToCompute`, receive a well-formed message...

Reply:
OK. I've added all, also I had to add few lines to `test_basic.py`